src/core/lib/iomgr/port.h

```diff
@@ -37,6 +37,7 @@
 #define GRPC_POSIX_SOCKETUTILS 1
 #define GRPC_POSIX_WAKEUP_FD 1
 #define GRPC_TIMER_USE_GENERIC 1
+#define GRPC_LINUX_LEGACY_EPOLL 1
```

Comment:
Can we just make this ```GRPC_LINUX_EPOLL```  instead of introducing a new variable?  We already have another flag for ```GRPC_LINUX_EPOLL_CREATE1```

Reply:
done